DragonSlayer.h

```diff
@@ -1,8 +1,19 @@
 #pragma once
+#include "Character.h"
+#include "AttackItem.h"
 
 struct DragonSlayer : Character
 {
+    DragonSlayer(std::string name_, int hp, int armor);
+    ~DragonSlayer () override;
+
+    const std::string& getName() override;
+    std::string getStats() override;
+
     void attack(Character& other) override;
+
+    AttackItem dragonSword;
```

Comment:
This needs to be resettable, like the other items. 
- [ ] store in a `unique_ptr<>`